packages/types/api/executions.d.ts

```diff
@@ -7,9 +7,10 @@ export type ExecutionProcessingTimes = {
 
 export interface ExecutionRecord {
```

Comment:
Should these types have a similar name, e.g. ApiExecutionRecord and ApiRecord for consistency with granules, pdrs, etc?

Reply:
Resolved https://github.com/nasa/cumulus/pull/3186/commits/05e4e6bec64eec060eeb397702228c8613031cb9